STAR/draft-ietf-acme-star.md

```diff
@@ -526,38 +524,39 @@ to, POST-as-GET), and a client to enable this service with per-Order
 granularity.
 
 Specifically, a server states its availability to grant unauthenticated access
-to a client's Order star-certificate by setting the star-allow-certificate-get
+to a client's Order star-certificate by setting the allow-certificate-get
 attribute to true in the meta field of the Directory object:
```

Comment:
in the `auto-renewal` object inside the meta field

Reply:
yep